repair/row_level.cc

```diff
@@ -612,6 +612,9 @@ class repair_meta {
     const repair_hash& working_row_buf_combined_hash() const {
         return _working_row_buf_combined_hash;
     }
+    bool use_rpc_stream() const {
+        return is_rpc_stream_supported(_algo);
+    }
 
```

Comment:
But you used it already, no?

Reply:
No. We will use rpc stream only if the _algo is row_level_diff_detect_algorithm::send_full_set_with_rpc_stream.

It is not enabled in this series.

Reply:
I meant I saw calls to `use_rpc_stream()` before.

Reply:
No, I checked. use_rpc_stream is used only since patch: "repair: Hook rpc stream version of verbs in row level repair".